index.bs

```diff
@@ -798,97 +930,103 @@ Reflective modules are always already instantiated.
 
-<ul>
-<li>syntax for accessing module local information: <code>import local from this;</code>
-<li>dynamic import: <code>local.import()</code>
-<li>extending the hooks to handle <code>this</code>
-<li>debugging info
-<li>room for host environment-specific data
-</ul>
+Besides the internal slots and the length property (whose value is 0), the Reflect.Loader constructor has the following properties:
 
+<h4 id="Reflect.Module.evaluate">Reflect.Module.evaluate(m)</h4>
 
-<h2 id="browser">Browser Loader</h2>
+<b>TODO:</b> way to force evaluation of a module namespace exotic object (<code>Reflect.Module.evaluate(m)</code>? <code>m[Reflect.Module.evaluate]()</code>?)
+
+<h4 id="Reflect.Module.prototype">Reflect.Module.prototype</h4>
+
+The value of Reflect.Module.prototype is an ordinary object with a null [[Prototype]].
 
-<h3 id="browser-site-packages">Site Packages</h3>
+This property has the attributes { \[[Writable]]: false, \[[Enumerable]]: false, \[[Configurable]]: false }.
 
-The browser loader contains extra properties for storing <dfn>site packages</dfn>, an application-global set of globally available packages. These map in an internal table to unique URLs that in turn serve as keys in the module registry.
+<h3 id="module-prototype-object">Properties of the Reflect.Module Prototype Object</h3>
 
-<div class="note">
-<p>
-The site package system serves as a simple coordination mechanism for modest-sized applications, but it does not provide all functionality required of a full-fledged package management system. It is expected that development ecosystems will build around package management tools that deal with requirements outside the scope of this specification, such as version management and allowing multiple versions of a library to coexist with the same name.
-</p>
+<h4 id="Reflect.Module.prototype.constructor">Reflect.Module.prototype.constructor</h4>
 
-<p>
-Tools that preprocess JavaScript source code may choose to use or ignore the site package table. For example, a package manager may choose to preprocess two separate import statements requiring <code>"jquery"</code> to <code>"jquery/1.9"</code> and <code>"jquery/2.1.1"</code> respectively, based on configuration files informing the tool of version requirements. The tool would then store both versions of jQuery in the site package table using the longer names. Alternatively, the tool may choose to preprocess the imports directly as URLs and bypass the site package system altogether.
-</p>
-</div>
+The initial value of Reflect.Module.prototype.constructor is Reflect.Module.
 
-The browser loader has an extra internal slot:
+<h3 id="reflect-module-internal-slots">Properties of Module Instances</h3>
+
+Reflect.Module instances are ordinary objects that inherit properties from the Reflect.Module.prototype.
+
+Reflect.Module instances are initially created with the internal slots described in the following table:
```

Comment:
This stuff shouldn't get moved out of the section on Reflective Module Records. The \[[LocalExports]], \[[IndirectExports]], and \[[Evaluate]] internal properties belong on a Reflective Module Record, which is encapsulated internally by the module namespace exotic object returned by the `Reflect.Module` constructor.